sources/common/src/SignalFactory.cpp

```diff
@@ -32,13 +32,16 @@
 #include "ISignalCreator.h"
 #include "FSignalCreator.h"
 
-PARSER::SignalFactory::SignalFactory() :
+PARSER::SignalFactory::SignalFactory(bool default_creators) :
```

Comment:
I don't like this.
Possible solution:
There should be a second class on the same level as `XmlSignalFactory` named for example `DefaultSignalFactory`. This class would register default creators. It should be created when user doesn't provide xml file.

Reply:
Good idea.